Cubical/Foundations/Path.agda

```diff
@@ -112,12 +116,30 @@ compPathr-isEquiv p = isoToIsEquiv (iso (_∙ p) (_∙ sym p) (compPathr-cancel
 compPathrEquiv : {x y z : A} (p : y ≡ z) → (x ≡ y) ≃ (x ≡ z)
 compPathrEquiv p = (_∙ p) , compPathr-isEquiv p
 
--- Variation of isProp→isSet for PathP
-isProp→isSet-PathP : ∀ {ℓ} {B : I → Type ℓ} → ((i : I) → isProp (B i))
+-- Variations of isProp→isSet for PathP
+isProp→SquareP : ∀ {B : I → I → Type ℓ} → ((i j : I) → isProp (B i j))
+             → {a : B i0 i0} {b : B i0 i1} {c : B i1 i0} {d : B i1 i1}
+             → (r : PathP (λ j → B j i0) a c) (s : PathP (λ j → B j i1) b d)
+             → (t : PathP (λ j → B i0 j) a b) (u : PathP (λ j → B i1 j) c d)
+             → SquareP B t u r s
+isProp→SquareP {B = B} isPropB {a = a} r s t u i j =
+  hcomp (λ { k (i = i0) → isPropB i0 j (base i0 j) (t j) k
+           ; k (i = i1) → isPropB i1 j (base i1 j) (u j) k
+           ; k (j = i0) → isPropB i i0 (base i i0) (r i) k
+           ; k (j = i1) → isPropB i i1 (base i i1) (s i) k
+        }) (base i j) where
+    base : (i j : I) → B i j
+    base i j = transport (λ k → B (i ∧ k) (j ∧ k)) a
+
+isProp→isPropPathP : ∀ {ℓ} {B : I → Type ℓ} → ((i : I) → isProp (B i))
                    → (b0 : B i0) (b1 : B i1)
                    → isProp (PathP (λ i → B i) b0 b1)
-isProp→isSet-PathP {B = B} hB b0 b1 =
-  transport (λ i → isProp (PathP≡Path B b0 b1 (~ i))) (isProp→isSet (hB i1) _ _)
+isProp→isPropPathP {B = B} hB b0 b1 = isProp→SquareP (λ _ → hB) refl refl
```

Comment:
`isProp→isSet-PathP` -> `isProp→isPropPathP` seemed more appropriate. In most cases suggested by `isSet`, one'd probably want to use `isProp→SquareP`.